packages/paste-website/src/pages/experiences/navigation/index.mdx

```diff
@@ -50,7 +51,7 @@ export const getStaticProps = async () => {
 
 ### General
 
-The Navigation pattern structures common elements found in Twilio products to provide customers a cohesive, expected way to find what they’re looking for. Applying this pattern across all our experiences also makes the Twilio product suite feel and behave like one family of products.
+The Navigation experience structures common elements found in Twilio products to provide customers a cohesive, expected way to find what they’re looking for. Applying this experience across all our experiences also makes the Twilio product suite feel and behave like one family of products.
```

Comment:
Dangit lol we used "experiences" to mean something else in the navigation docs. Let's update to:

> Applying this experience across all our customer experiences also makes the Twilio product suite feel and behave like one family of products.
> 
> What’s a product, customer experience, or feature?
> Products or product platforms: Twilio Console, Flex, Segment, and SendGrid.
> Customer experiences: All the products above, as well as Twilio Docs and internal tools.

Reply:
whoops clearly did not read that one over